package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.93",
+  "version": "1.1.97",
```

Comment:
So if you move here to bump the version in the PR then create the tag off master after merge?

Reply:
We err on the side of continuing to bump if we realize we need additional commits because yarn cache and Jenkins can get quite screwy otherwise. We can make a tag after the fact e.g. if you merge a PR, and then realize you forgot to tag it. Really, the only key is that there's no commits between the desired commit and the tag.

Not sure I understood your question or answered it :)

Reply:
mostly just making sure I understand when I need to do it.  I think we're run into the problem a few times in the past there was some confusion about when to create the tag.  It might be useful to have a script to bump the version and create a tag in the master branch just to make it stupid easy.